lenovo/thinkpad/default.nix

```diff
@@ -1,10 +0,0 @@
-{ lib, pkgs, ... }:
```

Comment:
This breaks every other configuration in this directory, since they all import this.